source/r/check_presence.R

```diff
@@ -35,23 +34,20 @@ check_presence <- function(
   for (country in countries) {
     # Function to check occurrence data and handle errors
     check_occurrence <- function(name) {
```

Comment:
I'm not a big fan of defining functions inside of other functions. 
Here its even a step further: the code defines a function in a for loop inside a function. Meaning you define the function for each iteration of the loop, which is just a bit weird to me ...